app/views/simple_discussion/forum_votes/_voting_button.js.erb

```diff
@@ -0,0 +1,4 @@
+$("a[data-remote]").off()
+$("a[data-remote]").on("ajax:success", function(event){
+  $(this).closest('.votable-container').html("<%= j render partial: 'shared/voting', locals: { user: current_user, votable: @votable } %>")
+});
```

Comment:
Could we convert these to vanilla Javascript? There's on jQuery dependency right now, so it's probably best to keep it that way.